.travis.yml

```diff
@@ -1,5 +1,5 @@
 language: php
-
+dist: trusty
```

Comment:
I didn't know that the `dist` key can be used within matrices inclusion. I thought only `env` and `php` was available.

So we need to do the same within the other repositories. See those PR which enable Trusty for all jobs:
symfony-cmf/routing-auto/pull/92
symfony-cmf/routing-auto-bundle/pull/203

We should think about a way to share a  common Travis configuration file for all repositories. This is becoming tedious to maintain its configuration for just 4 repositories. I can't image the headache after updating it for all Symfony CMF repositories.

It explains why those files are so different. Not only in there formatting. But also concerning the PHPUnit distribution type, its version,.